src/main/cpp/t07_swap.cpp

```diff
@@ -19,5 +19,21 @@
 using namespace std;
 
 int t07_swap() {
-
+	int n, c = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'c' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4674477)